docs/src/InvariantTheory/reductive_groups.md

```diff
@@ -4,18 +4,23 @@ CurrentModule = Oscar
 
 # Invariants of Linearly Reductive Groups
```

Comment:
I think we should make it clear somewhere that this part of the documentation is about experimental functionality. Either there could be some "warning" here or the files could be moved to the experimental directory (and we just link at them here).

In the end, this is all part of the discussion in #2011.

Reply:
@joschmitt, @fingolfin How about moving all this this from experimental to non-experimental part?

Reply:
The idea of experimental is (to my understanding) that people can get stuff in easy and that the interface is still allowed to change. I do not know what the status of the code is. I assume that names like `fundamental_invariants` etc. are set in stone but maybe other things will change?
The other question (to me) is, who will maintain and extend this code in the future, for example, to include more groups than SL_n and tori. I'm happy to do minor maintenance work, but I don't see myself working with linearly reductive groups.

Reply:
Okay, that's fine by me.